pkg/components/securedata/securedata.go

```diff
@@ -0,0 +1,26 @@
+package securedata
+
+import (
+	"encoding/base64"
+
+	"github.com/grafana/grafana/pkg/setting"
+	"github.com/grafana/grafana/pkg/util"
+)
+
+type SecureData string
+
+func EncryptAndEncode(data []byte) (SecureData, error) {
```

Comment:
Was thinking instead to have these functions as `Encrypt` and `Decrypt` and rename `SecureData` to `SecureEncodedData`. Opinions?